chapter3.Rmd

```diff
@@ -216,7 +216,7 @@ success_msg("Great! You're on the way of becoming an R jedi! Continue to the nex
 --- type:NormalExercise xp:100
 ## Calculating the worldwide box office 
 
-The single most important thing for a movie in order to become an instant legend in Tinseltown is its worldwide box office figures. 
+The single most important thing for a movie in order to become an instant legend in Tinseltown, is its worldwide box office figures. 
```

Comment:
I'd leave the comma off here. You can put it back in a new commit and the pull request will automatically update when you push to GitHub.